benchmarks/dynamo/check_accuracy.py

```diff
@@ -11,6 +11,7 @@
 flaky_models = {
     "yolov3",
     "gluon_inception_v3",
+    "detectron2_maskrcnn_r_101_c4",
```

Comment:
This only happens with AVX2 test?

Reply:
detectron2_maskrcnn_r_101_c4 has different accuracy status with different config in CI:
1. When running with avx512, it passed with freezing https://github.com/pytorch/pytorch/blob/main/benchmarks/dynamo/ci_expected_accuracy/cpu_inductor_torchbench_freezing_inference.csv#L22, but failed without freezing https://github.com/pytorch/pytorch/blob/main/benchmarks/dynamo/ci_expected_accuracy/cpu_inductor_torchbench_inference.csv#L21
2.  When running with freezing, it passed with avx512 but failed with avx2.

And I cannot reproduce the accuracy failure of this model on my local avx2 machine. In CI, It has the same failure msg as detectron2_maskrcnn_r_50_c4 discussed in https://github.com/pytorch/pytorch/issues/127073, suppose this model also has a numerical issue and mark it as flaky.